app/components/crate-row.hbs

```diff
@@ -6,10 +6,10 @@
           {{@crate.name}}
         </a>
       {{/let}}
-      {{#if @crate.defaultVersion}}
-        <span local-class="version" data-test-version>v{{@crate.defaultVersion}}</span>
+      {{#if (and @crate.default_version (not @crate.yanked))}}
+        <span local-class="version" data-test-version>v{{@crate.default_version}}</span>
```

Comment:
is this a behavior change? did we previously hide this for yanked crates?

Reply:
Previously, we filtered out yanked versions in mirage, ensuring that `max_stable_version` always refers to a non-yanked version as seen in https://github.com/rust-lang/crates.io/pull/9840/files#diff-1145f70fab74d9fb32b469538c364a5b4fb1c663447498e92ad6a2c348f21ed6. Therefore, I believe this change doesn't alter the behavior.

Reply:
Hmm, but on the other hand, you might be right. Since we don't guarantee this part in the API, this could potentially be a behavioral change.

Reply:
hmm, in this case, should we keep the `defaultVersion` and change to something like:
```js
  get defaultVersion() {
    if (this.default_version && !this.yanked) {
      return this.default_version;
    }
  }
```

Reply:
isn't `this.defaultVersion == this.default_version`?

Reply:
I think previously `defaultVersion` would be `undefined` if it was a **yanked** version. However, `default_version` could be a version string even if it's **yanked**?

Reply:
ohhh, I think you're right. I missed the `filter(versions::yanked.eq(false))` in https://github.com/rust-lang/crates.io/blob/1f3322a2dd526f3eb2881b66eeb4788b5078db0b/src/models/krate.rs#L479 🙈 

in that case we can keep it as-is :)

Reply:
yeah, probably won't hurt.